spartan/aztec-network/eth-devnet/docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       - "${PWD}/out/jwt-secret.hex:/genesis/jwt-secret.hex"
     environment:
       - HTTP_PORT=8545
+      - WS_PORT=8546
```

Comment:
`run-locally` was failing without this since https://github.com/AztecProtocol/aztec3-packages/issues/11922